### PR TITLE
Automatically flip material icons in RTL.

### PR DESCRIPTION
**Pull request by Hixie**
I'm not really sure how to test this without a reference test, since
eventually we want to move this to a font feature and thus the obvious
test (looking for the Transform widget) isn't really valid.

### CODE REVIEW

**Comment by Hixie** [comment]
Fixes https://github.com/flutter/flutter/issues/11368 (for now).

File: dev/tools/update_icons.dart [hunk 1]
```diff
@@ -28,6 +28,82 @@ const Map<String, String> kIdentifierRewrites = const <String, String>{
...
+  'backspace',
+  'battery_unknown',
+  'call_made',
+  'call_merge',
+  'call_missed',
+  'call_missed_outgoing',
+  'call_received',
+  'call_split',
+  'chevron_left',
+  'chevron_right',
```
Comment (abarth) [inline dev/tools/update_icons.dart]: Crazytown

**Comment by abarth** [comment]
LGTM

File: dev/tools/update_icons.dart (hunk 1 above)
Comment (Hixie) [reply dev/tools/update_icons.dart]: Yeah. I considered making new versions of the icons with more direction-agnostic names, and leaving these as non-flipped, but I was worried that that wouldn't be forward-compatible if they ever switch to doing this in the font.

Comment (Hixie) [reply dev/tools/update_icons.dart]: (If they decide not to change this in the font, then we can revisit.)

**Pull request merged**